### PR TITLE
std.format_housekeeping

### PR DESCRIPTION
**Pull request by burner**
* reactived outcommented unittests
* moved comment examples into unittest examples
* fixed indention

### CODE REVIEW

File: std/format.d [hunk 1]
```diff
@@ -4052,103 +4054,96 @@ here:
     const string[] widths = [ "", "0", "4", "20" ];
     const string[] precs = [ "", ".", ".0", ".4", ".20" ];
     const string formats = "sdoxXeEfFgGaA";
-  /+
```
Comment (JakobOvrum) [inline std/format.d]: Why was this commented out in the first place? What is the rationale for putting it back in?

Comment (burner) [inline std/format.d]: 1. I have no idea.
2. It tests if formattedWrite behaves as snprintf for "sdoxXeEfFgGaA" (which it should)

File: std/format.d [hunk 2]
```diff
@@ -342,24 +342,24 @@ $(I FormatChar):
     Floating point infinities are formatted as $(B inf) or
     $(B infinity) if the
     $(I FormatChar) is lower case, or $(B INF) or $(B INFINITY) if upper.
+*/
```
Comment (CyberShadow) [inline std/format.d]: Closing the DDoc here and reopening it afterwards will effectively remove all of the above from the generated documentation (see doc autotester). Curiously this only affects DDoc, not DDox. Probably a weird compiler bug?

**Comment by JackStouffer** [comment]
LGTM

**Comment by DmitryOlshansky** [comment]
LGTM too

**Comment by andralex** [comment]
alrighty then

**Comment by burner** [comment]
thanks

**Pull request merged**